tcl/org.testeditor.tcl.dsl.ui/src/org/testeditor/tcl/dsl/ui/quickfix/TclQuickfixProvider.xtend

```diff
@@ -149,5 +145,29 @@ class TclQuickfixProvider extends XbaseQuickfixProvider {
 		}
 	'''
 
-}
+	@Fix(TclSyntaxErrorMessageProvider.MISSING_TEST_DESCRIPTION)
+	def void fixMissingTestDescription(Issue issue, IssueResolutionAcceptor acceptor) {
+		val defaultTestDescription = '''
+			* test step description
+			'''
+		acceptor.accept(issue, "Add missing test description",
+			'''
+			Add a test description that will reference 
+			the respective test of the test specification
+			''', null) [ element, context |
+			if ((element instanceof TclModel) && ( (element as TclModel)?.test?.specification != null )) {
+				val list = getMissingTestSteps((element as TclModel).test)
+				if (list.empty) {
+					context.xtextDocument.replace(issue.offset, 0, defaultTestDescription)
+				} else {
+					context.xtextDocument.replace(issue.offset, 0, '''
+						* «list.head.contents.restoreString»
```

Comment:
We should consider the correct position here. Also, we can use the validation to put the restored string in the `data` so that we don't need to recalculate it here.